src/index.js

```diff
@@ -256,7 +271,18 @@ function validateSchema (config) {
       region: {type: 'string'},
       secrets: secretsArraySchema
     },
-    required: ['region', 'secrets'],
+    required: ['secrets'],
```

Comment:
if region is not required, which region will be picked for Security Manager? Without region Security Manager won't work. The only other region is from env.AWS_REGION var. Imo, this is not obvious for users of this tool and we should not remove region from required, otherwise they would get errors not clear of why secret doesn't come back